phpunit-plugin-bootstrap.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+require_once( __DIR__ . '/lib/antecedent/patchwork/Patchwork.php' );
```

Comment:
@insprintorob:

Instead of hard-coding the loading of Patchwork, how about instead we let this conditionally load a separate project-specific bootstrap file? For example:

```php
if ( file_exists( __DIR__ . '/../phpunit-plugin-bootstrap.project.php' ) ) {
    require_once( __DIR__ . '/../phpunit-plugin-bootstrap.project.php' );
}
```

And then in your project you can `require` Patchwork as needed. For that matter, you can also include Patchwork as a submodule (or composer) in your plugin, instead of adding it as a dependency for wp-dev-lib.